codeworld-requirements/src/CodeWorld/Requirements/Checker/Types.hs

```diff
@@ -66,16 +66,16 @@ module Requirements.Types where
     getStage (NotUsed _) = Parse
```

Comment:
I don't think you should do it in this pull request, but it seems worth reevaluating whether some of these could be moved to a later stage.  NotUsed seems like it would be more correct during renaming, for example.